### ISSUE TITLE
Some closure actions on elements are incorrectly detected as "interactive"

### ISSUE THREAD

**Issue by tim-evans** [rwjblue/ember-template-lint#242]
The following markup will cause `ember-template-lint` to raise the warning "Interaction added to non-interactive element" (which this is not):

```hbs
<div class='detail' scrollTop={{scrollTop}} onscroll={{action (mut scrollTop) value='target.scrollTop'}}>
</div>
```

**Comment by rwjblue** [comment]
We currently consider adding closure actions to any element as adding "interactivity" (which is why this is triggering the error you mentioned).  

We do currently opt-out for `<form onsubmit={{action ....}}>` [here](https://github.com/rwjblue/ember-template-lint/blob/4c94d9bc92b9367db5db3435e8060d16b221167d/lib/rules/lint-invalid-interactive.js#L46-L48), but I think it is going to be fairly hard to list all known properties for either a whitelist or blacklist...

Suggestions?

**Comment by tim-evans** [comment]
Thanks for the quick response!

Right now, I'm ignoring this on CodeClimate (marking it as invalid), but it would be nice if we were smart about some built-ins.

Maybe a blacklist for this is better than a whitelist? Looking at from https://developer.mozilla.org/en-US/docs/Web/Events, it looks like the following may be best to blacklist:

- `onfocus`
- `onblur`
- `onkeydown`
- `onkeypress`
- `onkeyup`
- `onmousedown`
- `onmouseup`
- `onauxclick`
- `onclick`
- `ondblclick`
- `ondragstart`
- `ondrag`
- `ondragenter`
- `ondragover`
- `ondragleave`
- `ondrop`

This will get quite hairy   when compounded with touch and pointer events (maybe check for `onmouse*` / `ontouch*` / `onpointer*`)?

I will be happy to implement this as well 😄

**Comment by rwjblue** [comment]
In general, I'd prefer a whitelist approach but I agree that may not be possible. I think we can swap to a blacklist and also update the rules config to allow it to be specified in userland config...

**Comment by tim-evans** [comment]
I do agree with the whitelist 👍  I'll see if I can do something with that :)
